src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1071,18 +1062,17 @@
             "contact": "(075) 523-6168"
           },
           "vice_mayor": {
-            "name": "MARK STEPHEN D. MEJIA",
-            "contact": "523-9624"
+            "name": "JOHNNY CABRERA"
           }
         },
         {
           "municipality": "Mangatarem",
           "zip_code": "2413",
           "mayor": {
-            "name": "RAMIL P. VENTENILLA"
+            "name": "JENSEN F. VIRAY-VENTENILLA"
           },
           "vice_mayor": {
-            "name": "MICHAEL MON R. PUNZAL"
+            "name": "RONNIE PALISOC"
```

```suggestion
            "name": "RONNIE C. PALISOC"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=27